docs/cvl/cvl2/migration.md

```diff
@@ -28,34 +51,104 @@ Run `python3 CVL1_to_CVL2.0_syntax_update.py --help` for further instructions.
 
 The migration script only handles simple cases, and is not guaranteed to work.
 Some manual work and adjustment may be needed after running the script. The
-script may also make odd mistakes. 
+script may also make odd mistakes.
 
 The script will attempt to make the following changes:
  - replace `sinvoke f(...)` with `f(...)`
  - replace `invoke f(...)` with `f@withrevert(...)`
- - replace `f(...).selector` with `sig:f(...).selector`
+ - replace `f(...)`.selector with `sig:f(...).selector`
  - ensure that rules start with `rule`
- - 
+ - replace `static_assert` with `assert`
+ - replace `static_require` with `require`
+ - add `;` to the end of `pragma`, `import`, `using`, and `use` statements
+ - add a `;` to the end of a methods block entry if it doesn't seem to continue to the next line
+ - add `function` to the beginning of a methods block entry
+ - add `external` to unsummarized methods block entries
+ - change `function f(...)` to `function _.f(...)` for summarized external functions
 
 In particular, as the script only consumes spec files, there are decisions that
 it cannot make, as they are based on the Solidity code. Some of those are
 listed here.
 
-## Method declarations
+## Step 3: Fix type errors
 
-- for `public` functions, the method can be summarized either externally or
-  internally. Annotate visibility of `external` or `internal`, and if you need
-  both, repeat the declaration in the `methods` block.
+This is a good time to try running `certoraRun` on your spec.  The command-line
+interface to `certoraRun` has not changed in CVL 2, so you should try to verify
```

Comment:
that's not entirely true. The default mode will be a new interface and you can opt-in to the old one (but only temporarily).
@rahav-certora is working on documentation. Usually it will not affect users, unless they were using `--settings` (We get rid of it).